python/letsql/backends/let/hotfix.py

```diff
@@ -212,3 +213,9 @@ def letsql_cache(self, storage=None):
 @property
 def ls(self):
     return LETSQLAccessor(self)
+
+
+# non method hotfix
+ibis.formats.pyarrow._from_pyarrow_types[pa.binary_view()] = (
```

Comment:
Is this overriding the binary view type ?

Reply:
no, its teaching ibis how to convert from the pyarrow type, without this we get an error